docs/features/workload-prioritization.rst

```diff
@@ -425,7 +425,7 @@ In order for workload prioritization to take effect, application users need to b
 
 Limits
 ======
-ScyllaDB is limited to 8 service levels, including the default one; this means you can create up to 7 service levels.
+ScyllaDB is limited to 9 service levels, including the default one; this means you can create up to 8 service levels.
```

Comment:
If we keep the service level change in this PR (I don't know if we should...) please fold this specific change into the earlier patch which did the other 8->9 changes.